src/components/decklist/decklist.tsx

```diff
@@ -78,6 +82,8 @@ export function Decklist(props: Props) {
     [t],
   );
 
+  const xpSum = computeXPSum(groups.sideSlots);
```

Comment:
The side deck is behind a `hasAdditional` boolean flag. We should probably move the computation inside that block and call it there directly:

```ts
extraInfos={`${computeXPSum(groups.sideSlots)} ${t("common.xp")}`}
```